pkg/cmd/pr/create/create.go

```diff
@@ -94,7 +94,7 @@ func NewCmdCreate(f *cmdutil.Factory, runF func(*CreateOptions) error) *cobra.Co
 		`),
 		Example: heredoc.Doc(`
 			$ gh pr create --title "The bug is fixed" --body "Everything works again"
-			$ gh pr create --reviewer monalisa,hubot
+			$ gh pr create --reviewer monalisa,hubot,myorg/devteam
```

Comment:
Could it be something like this, please?
```suggestion
			$ gh pr create --reviewer monalisa,hubot  --reviewer myorg/team-name
```
I feel such an example demonstrates that the `--reviewer` flag can be repeated, draws more attention to the team example, and shows how team names often have dashes in them. What do you think?

Reply:
Perfect!